src/app/modules/email-variables/email-variables.component.html

```diff
@@ -1,3 +1,49 @@
-<p align="center">
-  email-variables component
-</p>
+<div>
+    <a md-raised-button color=accent class="btn-pos" (click)="addVariable()">ADD NEW VARIABLE</a>
+</div>
+<div class="header">
+	<md-card>
+	<h3>User Variable(s)</h3>
+	<md-divider></md-divider>
+	<md-card-content>
+                <table class="list">
+                    <thead>
+                        <tr>
+                            <th>Variable code</th>
+                            <th>Variable value</th>
+                            <th>Delete</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr *ngFor="let variable of userVar">
+                            <td class="v-wid">{{variable.variableCode}}</td>
+                            <td class="txt-edit" (click)="edit(variable)"><p [innerHTML]="variable.variableValue | safeHtml"></p></td>
+                            <td class="del-wid">
+                                <md-icon class="red" (click)="delete(variable._id)">delete</md-icon>
+                            </td>
+                        </tr>
+                    </tbody>
+                </table>
+            </md-card-content>
+	</md-card>
+</div>
+<div class="header">
+	<md-card>
+	<h3>System Variable(s)</h3>
```

Comment:
right now System variable is just a simple list. there is no functionality for system variable in HR system.